docs/en/relying-party-solution.rst

```diff
@@ -288,7 +284,7 @@ The Relying Party issues the signed Request Object, where a non-normative exampl
   }
   .
   {
-    "scope": "eu.europa.ec.eudiw.pid.it.1 pid-sd-jwt:unique_id+given_name+family_name",
+    "scope": "eu.europa.ec.eudiw.pid.it.1 email",
```

Comment:
A very minor point: by the time the only optional parameter that we introduced in the PID Data Model is the user Tax  code ``tax_id_number``. Maybe it is better to use that otherwise, we have an inconsistency with what we reported in that section.

Reply:
Done, thx